common/common.go

```diff
@@ -429,13 +430,13 @@ func BlockIngestor(c *BlockCache, rep int) {
 		if err != nil {
 			Log.Fatal("bad getbestblockhash return:", err, result)
 		}
-		lastBestBlockHash, err := hex.DecodeString(hashHex)
+		lastBestBlockHashBE, err := hex.DecodeString(hashHex)
 		if err != nil {
 			Log.Fatal("error decoding getbestblockhash", err, hashHex)
 		}
 
 		height := c.GetNextHeight()
-		if string(lastBestBlockHash) == string(parser.Reverse(c.GetLatestHash())) {
+		if bytes.Equal(lastBestBlockHashBE, parser.Reverse(c.GetLatestHash())) {
```

Comment:
Using `bytes.Equal()` here has no functional difference; it's just more correct and efficient than converting both byte arrays to strings. (I made this change since I'm touching this line anyway for the variable rename.)